test/ext/chainrulescore.jl

```diff
@@ -0,0 +1,70 @@
+using Enzyme
+using Test
+using ChainRules
+using ChainRulesCore
+using LinearAlgebra
+using EnzymeTestUtils
+
+fdiff(f, x::Number) = autodiff(Forward, f, Duplicated, Duplicated(x, one(x)))[2]
+
+@testset "import_frule" begin
+    f1(x) = 2*x
+    ChainRulesCore.@scalar_rule f1(x)  (5*one(x),)
+    Enzyme.@import_frule typeof(f1) Any
+    @test fdiff(f1, 1f0) === 5f0
+    @test fdiff(f1, 1.0) === 5.0
+
+    # specific signature    
+    f2(x) = 2*x
+    ChainRulesCore.@scalar_rule f2(x)  (5*one(x),)
+    Enzyme.@import_frule typeof(f2) Float32
+    @test fdiff(f2, 1f0) === 5f0
+    @test fdiff(f2, 1.0) === 2.0
+
+    # two arguments
+    f3(x, y) = 2*x + y
+    ChainRulesCore.@scalar_rule f3(x, y)  (5*one(x), y)
+    Enzyme.@import_frule typeof(f3) Any Any    
+    @test fdiff(x -> f3(x, 1.0), 2.) === 5.0
+    @test fdiff(y -> f3(1.0, y), 2.) === 2.0
+
+    @testset "batch duplicated" begin 
+        x = [1.0, 2.0, 0.0]        
+        Enzyme.@import_frule typeof(Base.sort) Any
+
+        test_forward(Base.sort, Duplicated, (x, Duplicated))
+        # Unsupported by EnzymeTestUtils
```

Comment:
@sethaxen would it be possible to add support for duplicatednoneed and variants to enzymetestutils.

I'm going to approve/merge this without those, but it would be nice to enable later